backend/migration/qdrant_migration.py

```diff
@@ -0,0 +1,163 @@
+# This script migrates qdrant collection from one environment to another
+# Environment can be same qdrant instance or different qdrant instance
+"""
+How to run:
+
+python -m backend.migration.qdrant_migration \
+--source_backend_url http://localhost:8000 \
+--source_qdrant_url https://ps-qdrant-prathamesh-ws.devtest.truefoundry.tech \
+--source_collection_name creditcard \
+--destination_backend_url http://localhost:8000 \
+--destination_qdrant_url https://ps-qdrant-prathamesh-ws.devtest.truefoundry.tech \
+--destination_collection_name creditcopy
+
+Add --source_prefix & --destination_prefix if qdrant instance has prefix enabled
+"""
+
+
+import requests, argparse
+
+from backend.types import CreateCollectionDto, AssociateDataSourceWithCollection
+from backend.logger import logger
+
+from qdrant_client import QdrantClient
+from backend.migration.utils import migrate
+
+
+
+
+def migrate_collection(
+    source_backend_url: str,
+    source_collection_name: str,
+    source_qdrant_url: str,
+    source_prefix: str,
+    destination_backend_url: str,
+    destination_collection_name: str,  
+    destination_qdrant_url: str,
+    destination_prefix: str, 
+):
+    # fetch collection from source
+    collections = None
+    with requests.get(f"{source_backend_url.rstrip('/')}/v1/collections/") as r:
+        collections = r.json().get("collections")
+
+    if collections == None:
+        raise Exception("No collections found")
+
+    fetched_collection = None
+    for collection in collections:
+        if collection.get("name") == source_collection_name:
+            fetched_collection = collection
+            break
+
+    if fetched_collection == None:
+        raise Exception(f"Collection {source_collection_name} not found")
```

Comment:
Why not 
https://qdrant.github.io/qdrant/redoc/index.html#tag/collections/operation/get_collection 
?